cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsProcessor.java

```diff
@@ -0,0 +1,182 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.opencds.cqf.fhir.utility.Resources.newResource;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.hl7.fhir.instance.model.api.IPrimitiveType;
+import org.hl7.fhir.r4.model.CanonicalType;
+import org.hl7.fhir.r4.model.IdType;
+import org.hl7.fhir.r4.model.Measure;
+import org.hl7.fhir.r4.model.Organization;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.PrimitiveType;
+import org.hl7.fhir.r4.model.Resource;
+import org.opencds.cqf.fhir.api.Repository;
+import org.opencds.cqf.fhir.cr.measure.CareGapsProperties;
+import org.opencds.cqf.fhir.cr.measure.MeasureEvaluationOptions;
+import org.opencds.cqf.fhir.cr.measure.common.MeasureScoring;
+import org.opencds.cqf.fhir.cr.measure.constant.CareGapsConstants;
+import org.opencds.cqf.fhir.cr.measure.enumeration.CareGapsStatusCode;
+import org.opencds.cqf.fhir.cr.measure.r4.utils.R4MeasureServiceUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/*
+Care Gaps Processor houses construction of result body with input of different Result Bodies, such as Document Bundle vs non-document bundle
+ */
+public class R4CareGapsProcessor {
+
+    private static final Logger ourLog = LoggerFactory.getLogger(R4CareGapsProcessor.class);
+    private final Repository repository;
+    private final MeasureEvaluationOptions measureEvaluationOptions;
```

Comment:
No need for this field since you effectively assign the measureEvaluations argument directly to r4CareGapsBuilder